CMakeLists.txt

```diff
@@ -337,3 +339,24 @@ else()
     add_executable (nheko ${OS_BUNDLE} ${NHEKO_DEPS})
     target_link_libraries (nheko ${NHEKO_LIBS} Qt5::Multimedia)
 endif()
+
+# Install target
```

Comment:
Could you wrap this whole section in a `if (LINUX) ... endif(LINUX)` block to fix the build on macos? Any other fix is also welcome.